### PR TITLE
Update sensitive-information-type-entity-definitions.md

### PR DESCRIPTION
**Pull request by pratjain523**
@chrfox some updates as requested by fast track team to create clarity in content

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ description: "There are 200 sensitive information types that are ready for you t
 
 This article lists all sensitive information type entity definitions. Each definition shows what a DLP policy looks for to detect each type. To learn more about sensitive information types, see [Sensitive information types](sensitive-information-type-learn-about.md)
 
+> [!NOTE]
+> Mapping of confidence level (high/medium/low) with accuracy number (numeric value of 1 to 100)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md:29]:
```suggestion
> Mapping of confidence level (high/medium/low) with accuracy number (numeric value of 1 to 100):
```

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 2]
```diff
@@ -17661,20 +17667,16 @@ No
...
+- A keyword from `Keyword_ssn` is found.
 
 A DLP policy has medium confidence that it's detected this type of sensitive information if, within a proximity of 300 characters:
-- The function Func_unformatted_ssn finds content that matches the pattern.
-- A keyword from Keyword_ssn is found.
-
-A DLP policy has low confidence that it's detected this type of sensitive information if, within a proximity of 300 characters:
-- The function Func_randomized_formatted_ssn finds content that matches the pattern.
-- A keyword from Keyword_ssn is found.
+- The function  Func_unformatted_ssn` finds content that matches the pattern.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md:17674]:
```suggestion
- The function  `Func_unformatted_ssn` finds content that matches the pattern.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pratjain523, kindly commit the suggested minor changes. Many thanks.

**Pull request merged**